src/uct/base/uct_cm.c

```diff
@@ -111,6 +111,23 @@ void uct_cm_ep_server_conn_notify_cb(uct_cm_base_ep_t *cep, ucs_status_t status)
     cep->server.notify_cb(&cep->super.super, cep->user_data, &notify_args);
 }
 
+int uct_cm_set_listener_backlog(const uct_listener_params_t *params,
+                                int default_value)
```

Comment:
rename to `unsigned uct_listener_backlog_adjust(const uct_listener_params_t *params, unsigned max_value)` since it does not set anything
also `default_value` is actually `max_value` since you detect it explicitly for rdma_cm but for TCP it's:
"       If   the   backlog   argument   is    **greater**    than    the    value    in
       /proc/sys/net/core/somaxconn,  then it is **silently truncated to that value**;
       the default value in this file is 128.   In  kernels  before  2.4.25,  this
       limit was a hard coded value, SOMAXCONN, with the value 128.
" (c) man listen

Reply:
thanks for this comment! 
actually in kernel 3.10 it's already a greater value:
```
alinas@jazz04 ~
$cat /proc/sys/net/core/somaxconn
15000
alinas@jazz04 ~
$uname -r
3.10.0-957.el7.x86_64
```

so this means that with the new env param, we can only decrease the backlog value for tcp.

Reply:
seems like that and the only way to increase the backlog is system limits. (in case if UCX default values = max values)